sdks/python/apache_beam/internal/metrics/metric.py

```diff
@@ -223,3 +223,31 @@ def convert_to_canonical_status_string(self, status):
         http_status_code in http_to_canonical_gcp_status):
       return http_to_canonical_gcp_status[http_status_code]
     return str(http_status_code)
+
+  @staticmethod
+  def convert_to_grpc_status_string(grpc_status):
+    # type: (int) -> str
+
+    """Converts a grpc status to a canonical GCP status code string."""
```

Comment:
Replace this comment
"""
Converts the bigtable error code to a canonical GCP status code string.

This Bigtable client library is not using the canonical http status code values (i.e. https://cloud.google.com/apis/design/errors)"
Instead they are numbered using an enum with these values corresponding to each status code:
https://cloud.google.com/bigtable/docs/status-codes
"""

Please also add Args and Returns pydoc comment

Reply:
Fixed